_docker/drupal/dev/docker-compose.yml

```diff
@@ -1,7 +1,7 @@
 version: '2'
 services:
   update-composer:
-    image: images.paas.redhat.com/rhdp/developer-base:rhel-76.3
+    image: images.paas.redhat.com/rhdp/developer-base:rhel-76.4
```

Comment:
I need to understand if this change is alright based upon Rob Blake's recent PR to update the base image version elsewhere in the codebase

Reply:
Rob confirmed that these should be updated to 76.4